pkg/config/config.go

```diff
@@ -37,6 +38,28 @@ type Config struct {
 
 // Validate does static validation of the configuration
 func (c Config) Validate() error {
+	// Validate kernel version
+	// For valid versions see https://aquasecurity.github.io/tracee/latest/getting-started/installing/prerequisites/
+
+	// Default minimum kernel version
+	minimalKernelVersion := "5.4"
+	if c.OSInfo.GetOSReleaseID() == helpers.CENTOS {
+		// In centos tracee is supported in 4.18 kernel as well
+		minimalKernelVersion = "4.18"
+	}
+
+	res, err := c.OSInfo.CompareOSBaseKernelRelease(minimalKernelVersion)
+	if err != nil {
+		// TODO  what should we do?
```

Comment:
@rafaeldtinoco some suggestion?

Reply:
I don't think we should return an error if compare method itself failed.
Better to continue opportunistically in that case